examples/image-manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+  "kind": "image-manifest-v0",
+  "value": {
+      "bin": [
+          "/usr/sbin/cupsd"
+      ],
+      "service": [
+          "/lib/systemd/system/cups.service",
+          "/lib/systemd/system/printer.target.wants/"
```

Comment:
Just for reference: yes, the whole directory is copied. If it already exists, its content is merged in the existing one without overwriting existing assets.